src/features/account/MoreOptionMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro';
```

Comment:
this file will be used in the future when account-profile page PR was merged. this will be entry point to our `MoreOptionMenu` defined in figma.